lib/Compilation.js

```diff
@@ -1041,6 +1050,8 @@ class Compilation extends Tapable {
 	 * @returns {void} returns
 	 */
 	addEntry(context, entry, name, callback) {
+		this.hooks.addEntry.call(entry);
```

Comment:
rename to `buildEntry` and also pass `name` as arguments

Reply:
`addEntry` is not the same that `buildEntry`
`addEntry` will be called when adding an entry and `buildEntry` will be called when modules for an entry will be built
Or what do you mean? :)